Welcome stranger message can be improved in student home #1563

RatulGhosh opened the pull request:
Fixes #1563

damithc commented:
Um, that issues has a PR that is currently actively being discussed. So, it's best if we wait to see what happens to that PR before we invest in another one. :-)

RatulGhosh commented:
I am posting the screenshot
![capture](https://cloud.githubusercontent.com/assets/8985903/13517416/4feeaf5a-e1e9-11e5-8a1d-1d4ad88f0fd5.PNG)

RatulGhosh commented:
@damithc is it okay
![capture](https://cloud.githubusercontent.com/assets/8985903/13546542/d52370d2-e2d6-11e5-89ec-1d58a5ad6c1f.PNG)

RatulGhosh commented:
Sir, I have changed the message as suggested by you. I just used some different fonts and added some additional spaces to make it more readable. If you suggest I will change it back to the original look .

wkurniawan07 commented:
@RatulGhosh sorry for overlooking this, since #4870 has been closed, you are free to continue on this. You can learn from that PR to see what's expected.

RatulGhosh commented:
@wkurniawan07 I made some changes, could you please have a look

wkurniawan07 commented:
There are too many changes that are unrelated to the issue. I suspect it's because you didn't resolve the merge conflicts correctly. Can you do whatever repairs that are needed?

wkurniawan07 commented on src/main/java/teammates/common/util/Const.java:
```diff
@@ -896,18 +870,44 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
+													+ "<a href=" + ActionURIs.STUDENT_HOME_PAGE + ">Home</a> link above in "
+													+ "a few minutes."
+												+ "</li>"
+											+ "</ol>"
```
Use 4 spaces instead of tabs for indentation.

wkurniawan07 commented on src/test/resources/pages/studentHomeHTMLEmpty.html:
```diff
@@ -5,34 +5,40 @@
+ link above in a few minutes.
+</li>
+</ol>
+</div>
```
Please fix the HTML files with God mode instead of manually.

RatulGhosh commented:
I am getting some error while running God mode.
![capture](https://cloud.githubusercontent.com/assets/8985903/14541635/87748b18-02a8-11e6-8753-85ed9cf5bda3.PNG)

wkurniawan07 commented:
That is not a fault on God mode. Can you check the field `app.encryption.key` on your `build.properties` and make sure that there is no 'g' character in it, because `g` is not an acceptable base-16 character?

wkurniawan07 commented:
@RatulGhosh also, it'd be great if you can finish this PR by this milestone.

wkurniawan07 commented:
@RatulGhosh your change set here actually causes compile error and will not allow the application to build successfully. Please at least make sure that the application can run in your local machine before pushing for CI build.

wkurniawan07 commented:
If the unintended changes due to improper conflict resolution are too much to handle, you can close this PR and open another one with the intended changes only. By right, the change should only be on the designated constant in `Const.java` and some test files.

RatulGhosh commented:
Sir, I will try my best to finish it within a week, actually caught up with exams.

wkurniawan07 commented:
Closing due to inactivity.
